Doc/library/datetime.rst

```diff
@@ -1954,8 +1954,8 @@ Examples of working with a :class:`.time` object::
 
 .. method:: tzinfo.utcoffset(dt)
 
-   Return offset of local time from UTC, as a :class:`timedelta` object that is
-   positive east of UTC. If local time is west of UTC, this should be negative.
+   Returns the dfference between local time and UTC as a :class:`timedelta` object.
```

Comment:
```suggestion
   Return the dfference between local time and UTC as a :class:`timedelta` object.
```

See https://www.python.org/dev/peps/pep-0257/#id5 .

Reply:
Hmm, in the bpo it was suggested that the nonimperative  `Returns` can be used for documentation, though perhaps not docstring as the PEP is explicit about that. Would you still prefer to change all forms to `Return`?

Reply:
I personally like `Returns`, but everything else on [`datetime`](https://docs.python.org/3/library/datetime.html) uses `Return`, so let's be consistent with that for now and not let this PR get blocked on the question of what verb form to use.

I'll create a separate issue for switching from `Return` to `Returns`.

Reply:
I personally like `Returns`, but everything else on [`datetime`](https://docs.python.org/3/library/datetime.html) uses `Return`, so let's be consistent with that for now and not let this PR get blocked on the question of what verb form to use.

I'll create a separate issue for switching from `Return` to `Returns`.

Reply:
Thanks, I've updated the PR accordingly